### PR TITLE
Create item list

### PR DESCRIPTION
**Pull request by mayu3104**
add Dashboard section

### CODE REVIEW

File: public/js/app.js
Comment (Ditzler01) [inline public/js/app.js:1]: This file should not be included in PR.
This issue has been resolved and is already in the main branch.
Please update your main branch then merge the main branch to this branch to solve this issue.
Thank you.

File: public/js/dashboard.js
Comment (Ditzler01) [inline public/js/dashboard.js:1]: Same here.

Comment (mayu3104) [reply public/js/dashboard.js:1]: This js file is needed for dashboard page, how can I correct this file?

Comment (Ditzler01) [reply public/js/dashboard.js:1]: Every time you run [npm run dev] in the terminal, this file will be automatically created. That's why we don't include this to pull requests. Your main dashboard.js file is located in the resources folder, but this is from the public folder which is the compiled version of your dashboard.js from your resources folder.

File: resources/views/dashboard/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            <option value="">All</option>
+          </select>
+        </div>
+      </div>
+      <div class="col post rounded ms-1">
+        <p class="title">Post</p>
+        <div class="form-group">
+          <select class="form-control text-center fw-bold" id="FormSelect3">
+            <option value="" selected>All</option>
+            <option value="">All</option>
```
Comment (keanbillyfernandez) [inline resources/views/dashboard/index.blade.php:46]: is this a temporarily or just permanent values?

Comment (mayu3104) [reply resources/views/dashboard/index.blade.php:46]: This is a temporaly values.

File: public/js/dashboard.js
Comment (mayu3104) [reply public/js/dashboard.js:1]: You mean this dashboard.js file is copied from  resources folder, right?
So, how can I solve this?

**Pull request merged**